docs/toilAPI.rst

```diff
@@ -21,7 +21,7 @@ The Runner contains the methods needed to configure and start a Toil run.
 
 .. autoclass:: toil.job::Job.Runner
    :members:
-   
+
 Toil
```

Comment:
I noticed a formatting problem on this page when it's built. In the Job.run documentation, among other places, the 'Parameter' label overlaps the word 'fileStore'. 

Can you replicate that? I don't think it was introduced in this PR but if you can find a way to change/improve the resulting html format it would be awesome.